File: SingleAPIExample/python/VerifyPayment.py
import splitit

username, password, api_key = (
    "username",
    "password",
    "api_key",
)

api_client = splitit.ApiClient(splitit.Configuration(username, password, api_key, sandbox=True))


def verify_payment(ipn):
    resp = api_client.InstallmentPlanApi.installment_plan_verify_payment(
        splitit.VerifyPaymentRequest(
            installment_plan_number=ipn,
        )
    )
    return resp

File: SingleAPIExample/java/VerifyPayment.java
import com.splitit.*;
import com.splitit.sdk.model.*;
import java.math.BigDecimal;


public class VerifyPayment {
    private String apiKey, username, password;
    public VerifyPayment(String username, String password, String apiKey){
        this.username = username;
        this.password = password;
        this.apiKey = apiKey;
    }

    public void example(String ipn) throws ApiException{
        ApiClient apiClient = new ApiClient()
            .sandbox(true)
            .username(username)
            .password(password)
            .apiKey(apiKey);


        var result = apiClient.getInstallmentPlanApi().installmentPlanVerifyPayment(
            new VerifyPaymentRequest()
            .installmentPlanNumber(ipn)
        );
        System.out.println(result);
    }
}
